wordpress-mobile/gutenberg-mobile
# Add GlotPress project slug parameter to translations pipeline

**Related PRs:**
* https://github.com/WordPress/gutenberg/pull/49558

Related to https://github.com/wordpress-mobile/gutenberg-mobile/issues/5622.

**To test:**
1. Run command `npm run i18n:update`.
2. Observe that the command runs successfully.
3. Observe that localization strings files (`bundle/android/strings.xml` and `bundle/ios/GutenbergNativeTranslations.swift`) remain unmodified. Or if they are modified, that the modification is expected and produced by recent changes coming from Gutenberg web version.
4. Test the editor in a non-English language and check that strings are translated as expected.

PR submission checklist:
